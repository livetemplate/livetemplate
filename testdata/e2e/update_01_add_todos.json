{"s":["<div class=\"container\">\n        <header>\n            <h1>","</h1>\n            <div class=\"counter\">Count: ","</div>\n            <div class=\"status ","\">\n                Status: ","\n            </div>\n        </header>\n        \n        <main>\n            <section class=\"stats\">\n                <h2>Statistics</h2>\n                <p>Total Todos: ","</p>\n                <p>Completed: ","</p>\n                <p>Remaining: ","</p>\n                <p>Completion Rate: ","</p>\n            </section>\n            \n            <section class=\"todos\">\n                <h2>Todo List</h2>\n                ","\n            </section>\n        </main>\n        \n        <footer>\n            <p>Last updated: ","</p>\n            <p>Session ID: ","</p>\n        </footer>\n    </div>"],"0":"Task Manager","1":"3","2":"inactive","3":"Low Activity","4":"3","5":"1","6":"2","7":"33%","8":{"d":[{"0":"","1":"k1pskw","2":"#0","3":"Learn Go templates","4":"○","5":"(Priority: High)"},{"0":"completed","1":"k1142q","2":"#1","3":"Build live updates","4":"✓","5":"(Priority: Medium)"},{"0":"","1":"k1aqr4","2":"#2","3":"Write documentation","4":"○","5":"(Priority: Low)"}],"s":["\n                    <div class=\"todo-item ","\" data-lvt-key=\"","\">",">\n                        <strong>",":</strong> \n                        "," \n                        ","\n                        ","\n                    </div>\n                    "]},"9":"2023-01-01 10:15:00","10":"session-12345"}