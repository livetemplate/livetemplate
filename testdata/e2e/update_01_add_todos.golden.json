{
  "0": "Task Manager",
  "1": "3",
  "2": "inactive",
  "3": "Low Activity",
  "4": "3",
  "5": "1",
  "6": "2",
  "7": "33%",
  "8": {
    "d": [
      {
        "0": "1",
        "1": "",
        "2": "#0",
        "3": "Learn Go templates",
        "4": "○",
        "5": "(Priority: High)"
      },
      {
        "0": "2",
        "1": "completed",
        "2": "#1",
        "3": "Build live updates",
        "4": "✓",
        "5": "(Priority: Medium)"
      },
      {
        "0": "3",
        "1": "",
        "2": "#2",
        "3": "Write documentation",
        "4": "○",
        "5": "(Priority: Low)"
      }
    ],
    "s": [
      "<div data-lvt-key=\"",
      "\">\n                    <div class=\"todo-item ",
      "\">\n                        <strong>",
      ":</strong> \n                        ",
      " \n                        ",
      "\n                        ",
      "\n                    </div>\n                    </div>"
    ]
  },
  "9": "2023-01-01 10:15:00",
  "10": "session-12345",
  "s": [
    "<div class=\"container\">\n        <header>\n            <h1>",
    "</h1>\n            <div class=\"counter\">Count: ",
    "</div>\n            <div class=\"status ",
    "\">\n                Status: ",
    "\n            </div>\n        </header>\n        \n        <main>\n            <section class=\"stats\">\n                <h2>Statistics</h2>\n                <p>Total Todos: ",
    "</p>\n                <p>Completed: ",
    "</p>\n                <p>Remaining: ",
    "</p>\n                <p>Completion Rate: ",
    "</p>\n            </section>\n            \n            <section class=\"todos\">\n                <h2>Todo List</h2>\n                ",
    "\n            </section>\n        </main>\n        \n        <footer>\n            <p>Last updated: ",
    "</p>\n            <p>Session ID: ",
    "</p>\n        </footer>\n    </div>"
  ]
}